src/test/java/com/jcabi/log/MulticolorLayoutTest.java

```diff
@@ -167,4 +158,146 @@ public void throwsOnIllegalColorName() throws Exception {
         layout.format(event);
     }
 
+    /**
+     * MulticolorLayout disables default color
+     * when -Dcom.jcabi.log.coloring=false.
+     * @throws Exception - if something goes wrong.
+     */
+    @Test
+    public void disablesDefaultColor() throws Exception {
+        final MulticolorLayout layout = new MulticolorLayout();
+        layout.setConversionPattern(MulticolorLayoutTest.CONV_PATTERN);
+        final LoggingEvent event = Mockito.mock(LoggingEvent.class);
+        Mockito.doReturn(Level.DEBUG).when(event).getLevel();
+        Mockito.doReturn("no color").when(event).getRenderedMessage();
+        MatcherAssert.assertThat(
+            StringEscapeUtils.escapeJava(layout.format(event)),
+            Matchers.equalTo(
+                "[\\u001B[2;37mDEBUG\\u001B[m] \\u001B[2;37mno color\\u001B[m"
+            )
+        );
+        System.getProperties().setProperty(
+            COLORING_PROPERTY, Boolean.FALSE.toString()
+        );
+        try {
+            MatcherAssert.assertThat(
+                StringEscapeUtils.escapeJava(layout.format(event)),
+                Matchers.equalTo(
+                    "[DEBUG] no color"
+                )
+            );
+        } finally {
+            System.getProperties().setProperty(
+                COLORING_PROPERTY, Boolean.TRUE.toString()
+            );
+        }
+    }
+
+    /**
+     * MulticolorLayout disables the color that overwrites the default one.
+     * @throws Exception - if something goes wrong.
+     */
+    @Test
+    public void disablesOverridenDefaultColor() throws Exception {
+        final MulticolorLayout layout = new MulticolorLayout();
+        layout.setConversionPattern("[%color{%p}] %m");
+        layout.setLevels("DEBUG:2;10");
+        final LoggingEvent event = Mockito.mock(LoggingEvent.class);
+        Mockito.doReturn(Level.DEBUG).when(event).getLevel();
+        Mockito.doReturn("no colour text").when(event).getRenderedMessage();
+        MatcherAssert.assertThat(
+            StringEscapeUtils.escapeJava(layout.format(event)),
+            Matchers.equalTo(
+                "[\\u001B[2;10mDEBUG\\u001B[m] no colour text"
+            )
+        );
+        System.getProperties().setProperty(
+            COLORING_PROPERTY, Boolean.FALSE.toString()
+        );
+        try {
+            MatcherAssert.assertThat(
+                StringEscapeUtils.escapeJava(layout.format(event)),
+                Matchers.equalTo(
+                    "[DEBUG] no colour text"
+                )
+            );
+        } finally {
+            System.getProperties().setProperty(
+                COLORING_PROPERTY, Boolean.TRUE.toString()
+            );
+        }
+    }
+
+    /**
+     * MulticolorLayout disables constant color
+     * when -Dcom.jcabi.log.coloring=false.
+     * @throws Exception - if something goes wrong.
+     */
```

Comment:
@amihaiemil you have to create a new puzzle to fix the bug and to enable the tests